packages/sign_in_with_apple/sign_in_with_apple/darwin/Classes/SignInWithAppleError.swift

```diff
@@ -107,15 +107,21 @@ public enum SignInWithAppleError {
                 errorCode = "authorization-error/notHandled"
             case .failed:
                 errorCode = "authorization-error/failed"
-#if (os(iOS) && swift(>=5.5)) || (os(macOS) && swift(>=5.5.1))
+#if compiler(>=5.5)
             // new case since Xcode 13, arrived earlier in iOS
-            // use https://swiftly.dev/swift-versions to match Swift to Xcode versions (as this is in practice driven by the OS SDK, not Swift version)
+            // use https://xcodereleases.com/ to match Swift to Xcode versions (as this is in practice driven by the OS SDK, not Swift version)
             case .notInteractive:
                 errorCode = "authorization-error/notInteractive"
 #endif
 #if compiler(>=6.0)
             case .matchedExcludedCredential:
                 errorCode = "authorization-error/matchedExcludedCredential"
+#endif
+#if compiler(>=6.0.3)
+            case .credentialImport:
+                errorCode = "authorization-error/credentialImport"
```

Comment:
We should map these on the Dart side, as in https://github.com/aboutyou/dart_packages/blob/63b49af2e8cef437a2b4e2bff5adbd5dbe95a86a/packages/sign_in_with_apple/sign_in_with_apple_platform_interface/lib/exceptions.dart#L42

Now I also noticed that the already existing case (in Swift) of `matchedExcludedCredential` seems to be missing there.

Reply:
I have added the missing matchedExcludedCredential as wel as the other exceptions to the dart class.